### DISCUSSION TITLE
Signal Detection Help

### DISCUSSION THREAD

**Discussion by NjayNL** [awawa-dev/HyperHDR#843] [Q&A] [answered]
Hi All,

My setup: usb grabber ms2109 / hue lights running on the vero4k+ with the latest hyperhdr. The issue I am having is that the leds are pretty dim. Only on flashes and gun fire do they light up. Source movie: predators. It’s a dark movie I know, but is there any I can do ? Like adjusting signal detection?

**Comment by awawa-dev** [comment]
Hi
Are you using HDR signal and enabled tone mapping in HyperHDR? Verify video preview first, not LEDs. You also need to use correct LUT table for ms2109 and adjust ms2109's brightness/contrast/saturation to 1/128/128 because ms2109 factory default settings are not neutral.

**Reply by NjayNL** [reply]
It’s hardware brightness -11 the 148 / 180. If defaulting what the LuT programmed 148/148 the leds where very dim. Or do I need the brightness on 1 instead of - 11 ?

**Reply by awawa-dev** [reply]
https://www.hyperhdr.eu/2022/04/usb-grabbers-hdr-to-sdr-quality-test.html#chapter31

**Comment by NjayNL** [comment]
Good read! Had bad settings. Let me test tonight.

**Reply by awawa-dev** [reply]
Do not forget about LUT: v20 has built-in downloader and should automatically set brightness etc for ms2109. I assume that you are using HDR10 signal...

**Reply by NjayNL** [reply]
I grabbed the correct LUT. But manually adjusting settings helped. Any tips on resolution setting and signal detection and frame rate? I have it on 50 and 720p with no signal detection selected.

**Reply by awawa-dev** [reply]
from: https://github.com/awawa-dev/HyperHDR/wiki/Quick-start
`Use the highest framerate for your USB grabber. GENERAL RULE: HIGHEST FRAMERATE, LOWEST RESOLUTION.`
ms2109 usually has a rainbow no signal board so automatic no-signal detection is recommended (you can find more about it in v17 release info on the blog)

**Comment by NjayNL** [comment]
Almost there. Colors are good when watching a 4K hdr movie the only thing is that the live video
Is darker then the real screen. Which setting increases this?

**Comment by awawa-dev** [comment]
Not sure which colorspace your video player is using: you can calibrate LUT using HyperHDR for exactly the same signal and resolution ex 4k60 HDR 422 limited. Or try 2 different LUTs: the built one (delete custom LUT from home folder if you've installed it) or use LUT for ms2130 **limited** colorspace to see if it's better.

**Reply by NjayNL** [reply]
The colours are correct though it’s darker only. Which setting increases brightness ? And I will try the other lut’s too :)

**Reply by awawa-dev** [reply]
That's not that simple for HDR signal. You cannot increase brightness on the grabber because you will break the signal (only SDR can be adjusted this way)...so another LUT or calibration. Or just increase LED brightness (lower gammas)  in the processing tab but the preview video wont change.

**Reply by NjayNL** [reply]
First off, thanks for all the support. So i am getting the hang of it a little. i was able to use the ms2109 lut and adjust to 11/145/198 so it matches the output better. this also increases the brightness to my liking. now for the final steps. led layout. i have 2 hue leds strips on the bottom side of my media cabinet one on the front and one on the back. then 2 hue go's on the left and right side of the cabinet next to the tv. What would be the best settings when choosing the lights from the config. my guess the strips as bottom and the go's as left and right 25 to 75 from top?

**Reply by awawa-dev** [reply]
These are just initial settings. You can later adjust your lamps as you like: https://github.com/awawa-dev/HyperHDR/wiki/Philips-Hue-v2 at the end.

**Reply by NjayNL** [reply]
And my final question is ; if there are more colors then one in led layout section of the hue go lamp it tries to mix them? Because it’s one light and not individual leds like on a strip?

**Reply by awawa-dev** [reply, marked as answer]
will mixed the pixel in the selected area. Also LEDs of the LED strip have usually mixed colors because their areas usually have dozens of pixels depending on the geometry design and the size of the video frame.

**Reply by NjayNL** [reply]
switching to hue v2 api solved a lot :) i think i have it now! time to stick another hue led strip to the back of the tv for top! thank you so much. case closed
